core/frontend/src/tools/ClipViewTool.ts

```diff
@@ -60,7 +60,7 @@ export interface DrawClipOptions {
   id?: string;
 }
 
-/** @alpha A tool to define a clip volume for a view */
+/** @public A tool to define a clip volume for a view */
 export class ViewClipTool extends PrimitiveTool {
   constructor(protected _clipEventHandler?: ViewClipEventHandler) { super(); }
 
```

Comment:
Are protected members like `_orientationName` supposed to be `@public`?

Reply:
done (to keep these out of the docs)